core/domain/platform_parameter_list.py

```diff
@@ -112,6 +114,11 @@ class ParamNames(enum.Enum):
     'in the curriculum admin view.',
     platform_parameter_domain.FeatureStages.DEV)
 
+Registry.create_feature_flag(
+    ParamNames.SHOW_REDESIGNED_LEARNER_DASHBOARD,
+    'This flag is to show redesigned learner dashboard.',
+    platform_parameter_domain.FeatureStages.PROD)
```

Comment:
This starts at Dev. Please read https://github.com/oppia/oppia/wiki/Developing-new-features-with-feature-gating#how-do-you-as-a-developer-use-feature-flags carefully.

Reply:
Hi, @seanlip Thanks for the review. I am bit confused with the third bullet point [here](https://github.com/oppia/oppia/wiki/Developing-new-features-with-feature-gating#feature-stage-explanation).
Can you please take a moment to check it and let me know.
Thanks!

Reply:
@SubhashThenua It's hard to answer your question without knowing what exactly your point of confusion is. Next time please ask a more specific question.

Also, I'm on vacation so I only check PRs once a day. I suggest consulting with your mentors and particularly @kevintab95 if you have questions about the release process. Nevertheless if you follow the instructions at https://github.com/oppia/oppia/wiki/Developing-new-features-with-feature-gating#how-do-you-as-a-developer-use-feature-flags you should be fine.

Reply:
Hi, @kevintab95 I am bit confuse with the third bullet point [here](https://github.com/oppia/oppia/wiki/Developing-new-features-with-feature-gating#feature-stage-explanation).
The third point says **prod feature can be enabled in any of the dev, test, production environments.**.
I think If we use the feature stage as **PROD** then we did not need to make the other PRs to make it enable for the test,dev. As the 3rd point [here](https://github.com/oppia/oppia/wiki/Developing-new-features-with-feature-gating#how-to-use-feature-flags) says `The very last PR you make (to finish up the feature you are working on) must include changes that move the feature flag to the TEST stage. This is to ensure that the feature is available in the test environment

Thanks

Reply:
Just to circle back on this: the whole point of feature gating is to make the feature **inaccessible** in environments for which it is not ready. Thus, marking this feature as being in "dev stage" stops it from being enabled by accident on the prod servers.